tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -114,18 +114,6 @@ def __init__(self, env_name: str, test_cases: List[str]):
         super().__init__(msg)
 
 
-class _TestNode(object):
```

Comment:
The AgentTestSuite used a static list of test nodes and their IP addresses. I removed that list and now this information can be retrieved, as well as refreshed, from the AgentTestContext.